hazelcast/src/test/java/com/hazelcast/multimap/LocalMultiMapStatsTest.java

```diff
@@ -378,4 +371,20 @@ public void testLockedEntryCount_multiMapWithOneEntry() {
         LocalMapStats stats = getMultiMapStats();
         assertEquals(2, stats.getLockedEntryCount());
     }
+
+    @Test
+    @Ignore
+    public void testHitCountNotLost_whenKeysAreRemoved() {
+        MultiMap<Integer, Integer> map = getMultiMap();
+        for (int i = 0; i < 100; i++) {
+            map.put(i, i);
+            map.get(i);
+        }
+        for (int i = 0; i < 50; i++) {
+            map.remove(i);
+        }
+
+        LocalMapStats localMapStats = getMultiMapStats();
+        assertEquals(100, localMapStats.getHits());
+    }
```

Comment:
Can we reference the related GH issue in `@Ignore`?

Also, the tests in line 107 and 126 seem to miss the `@Test` annotation (not changed in this PR, just noticed).

Reply:
Linked the issue. The methods mentioned as the missing test annotations are helper methods which are called from another tests, I just changed the access modifiers of them so that they can be distinguished.